docs/design_guide.rst

```diff
@@ -269,6 +292,58 @@ Renders as:
     :param ~busio.I2C i2c_bus: The I2C bus the DS3231 is connected to.
     :param int address: The I2C address of the device. Defaults to :const:`0x40`
 
+
+Parameters Documentation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Although there are different ways to document class and functions definitions in Python,
+the following is the prevalent method of doing parameters documentations
+for CircuitPython libraries. When documenting class parameters you should use the
+following structure:
+
+.. code-block:: sh
+
+    :param param_type param_name: Parameter_description
+
+
+param_type
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Type of the parameter. This could be among other `int`, `float`, `str` `bool`, etc.
```

Comment:
This change is not necessary, but is worth considering.
```suggestion
The type of the parameter. This could be among other `int`, `float`, `str` `bool`, etc.
```